src/libproc_macro/lib.rs

```diff
@@ -130,20 +141,27 @@ impl fmt::Debug for TokenStream {
     }
 }
 
+/// Creates a token stream containing a single token tree.
+///
+/// REVIEW We don't generally have impls `From<T> for Collection<T>`, but I see why this exists
+/// REVIEW from practical point of view.
```

Comment:
Yeah this is currently the main way to go from a `TokenTree` to a `TokenStream`, and I think we also accidentally stabilized it already so we're required to keep it.